### ISSUE TITLE
[Victory]Multiple active match option for rules(even with maxCount=1)

### ISSUE THREAD

**Issue by Ayurshi** [eMoflon/emoflon-ibex-ui#146]
Rules with maxCount=1 should have active "Apply match" only once. After they are applied once, they should no longer have active apply match option. 
Currently they have active apply option more than once(possibly twice) even with maxCount=1.

**Comment by LegionaryCohort** [comment]
Duplicate of #144
